tests/cupy_tests/sorting_tests/test_sort.py

```diff
@@ -14,17 +14,19 @@ class TestSort(unittest.TestCase):
 
     # TODO(niboshi): Fix xfail
     @pytest.mark.xfail(reason='Explicit error types required')
-    @testing.numpy_cupy_raises()
-    def test_sort_zero_dim(self, xp):
-        a = testing.shaped_random((), xp)
-        a.sort()
+    def test_sort_zero_dim(self):
+        for xp in (numpy, cupy):
+            a = testing.shaped_random((), xp)
+            with pytest.raises(numpy.AxisError):
```

Comment:
Could you remove the involving `xfail`s in a new PR first?
- See https://github.com/cupy/cupy/pull/2870#discussion_r361290474 for the reason of `xfail`.
- CuPy `master` requires `numpy>=1.15`. (#2900)

Reply:
Splitted into #3167.